iib/workers/tasks/build_merge_index_image.py

```diff
@@ -128,28 +130,26 @@ def _add_bundles_missing_in_source(
             '%s bundles have invalid version label and will be deprecated.', len(invalid_bundles)
         )
 
-    with set_registry_token(overwrite_target_index_token, source_from_index):
+    with set_registry_token(overwrite_target_index_token, target_index, append=True):
         is_source_fbc = is_image_fbc(source_from_index)
-    if is_source_fbc:
-        opm_registry_add_fbc(
-            base_dir=base_dir,
-            bundles=missing_bundle_paths,
-            binary_image=binary_image,
-            from_index=source_from_index,
-            container_tool='podman',
-            overwrite_from_index_token=overwrite_target_index_token,
-        )
-    else:
-        _opm_index_add(
-            base_dir=base_dir,
-            bundles=missing_bundle_paths,
-            binary_image=binary_image,
-            from_index=source_from_index,
-            overwrite_from_index_token=overwrite_target_index_token,
-            # Use podman until opm's default mechanism is more resilient:
-            #   https://bugzilla.redhat.com/show_bug.cgi?id=1937097
-            container_tool='podman',
-        )
+        if is_source_fbc:
```

Comment:
Could not be indentation here be reduced by one tab (as in build.py case)?

Reply:
I need to have set the token correctly for these calls (is_image_fbc, opm_registry_add_fbc, _opm_index_add).
That is why it is indented this way.